.babelrc

```diff
@@ -4,7 +4,9 @@
       "env",
       {
         "targets": {
-          "browsers": ["last 2 versions"],
+          "browsers": [
```

Comment:
I am not sure if we can modify this if we want to merge back to original ledgerjs repository